closure/goog/net/jsloader.js

```diff
@@ -122,15 +134,17 @@ goog.net.jsloader.loadMany = function(uris, opt_options) {
     if (uris.length) {
       deferred.addBoth(popAndLoadNextScript);
     }
+    return deferred;
   };
-  popAndLoadNextScript();
+  goog.net.jsloader.scriptLoadingDeferred_ = popAndLoadNextScript();
+  return goog.net.jsloader.scriptLoadingDeferred_;
```

Comment:
Can you add a unit test that validates the deferred corresponds to the completion of loading all scripts?